app/frontend/components/pages/receive_ada/receiveAdaPage.js

```diff
@@ -11,8 +9,8 @@ const OwnAddressesList = connect('ownAddressesWithMeta')(({ownAddressesWithMeta}
     h('h2', undefined, 'My Addresses'),
     ...ownAddressesWithMeta.map((adr) =>
       h(AddressContainer, {address: adr.address, bip32path: adr.bip32StringPath})
-    ),
-    h(AddressDetailDialog)
+    )
+    // h(AddressDetailDialog) not needed anymore, as it is invoked from footer
```

Comment:
then it would be probably more suitable to move the AddressDetailDialog to some higher-level scope, like components/app.js and have this dialog between "common" components